src/main/java/com/minecolonies/blockout/views/View.java

```diff
@@ -46,18 +46,17 @@ public View(final PaneParams params)
         return children;
     }
 
-    @Override
-    public void parseChildren(final PaneParams params)
+    public void parseChildren(final IPaneParams params)
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add the "@Override" annotation above this method signature [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1161)